StyleCop.Analyzers/version.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "1.1.0-beta.{height}",
+  "version": "1.1.1-beta.{height}",
   // The build number offset accounts for the delay in adopting Nerdbank.GitVersioning
-  "buildNumberOffset": 40,
+  "buildNumberOffset": 56,
```

Comment:
Since you force pushed, github won't let me see the rest of the context of this change. But the buildNumberOffset is simply added to the value from the height calculation, whatever that is. The height is reset when you change a major.minor component only -- not the 3rd component. This is an oversight from when we added support for moving the height integer to someplace other than the 3rd component. This is tracked by https://github.com/AArnott/Nerdbank.GitVersioning/issues/198

Reply:
> Since you force pushed, github won't let me see the rest of the context of this change

This comment is the complete context. 😄